qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -131,29 +207,35 @@ def __init__(
             link_parameters: if set to True then the amplitude and DRAG parameters of the
                 X and Y gates will be linked as well as those of the SX and SY gates.
         """
-        super().__init__(basis_gates, default_values)
         self._link_parameters = link_parameters
+        self._use_drag = use_drag
+
+        extra_kwargs = {"use_drag": use_drag, "link_parameters": link_parameters}
```

Comment:
Why we do we need to define dictionary? This can be directly passed to constructor.

Reply:
We could, but this syntax better shows the intention of the `extra_kwargs` and emphasizes that they are passed to super init for serialization purposes.